bonus.js

```diff
@@ -0,0 +1,72 @@
+const express = require("express")
+const app = express()
+const cors = require("cors")
+
+let users = [
+    {
+        "username": "user6",
+        "password": "pw6",
+        "id": "16298100251"
+      }
+]
+app.use( cors())
+app.use( express.json())
+
+/// POST
+
+app.post("/users", (req, res) =>{
+    console.log("[POST] /users called")
+    const { username, password} = req.body
+
+    const userFound = users.find(user => user.username === username)
+    const passwordFound =users.find(user => user.password === password)
+    if(!userFound){
+        return res.json({
+            error: "Failed to login, user was not found!"
```

Comment:
Prob that was the issue, you kind of mixed the signup route with a login check :)